configs/components/rubygem-win32-dir.rb

```diff
@@ -0,0 +1,27 @@
+component "rubygem-win32-dir" do |pkg, settings, platform|
+  pkg.version "0.4.9"
+  pkg.md5sum "df14aa01bd6011f4b6332a05e15b7fb8"
+  pkg.url "http://buildsources.delivery.puppetlabs.net/win32-dir-#{pkg.get_version}.gem"
+
+  pkg.build_requires "ruby"
+
+  pkg.environment "PATH" => "#{settings[:bindir]}:$$PATH"
+
+  # Because we are cross-compiling on sparc, we can't use the rubygems we just built.
+  # Instead we use the host gem installation and override GEM_HOME. Yay?
```

Comment:
It's still nice to have an explanation of why this variable is set in the first place, when logically it shouldn't be necessary. If you feel strongly about removing it, I will.